tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -189,14 +184,40 @@ var _ = Describe("Verify Create", Ordered, func() {
 			Eventually(func(g Gomega) {
 				pods, _ := e2e.ParsePods(kubeConfigFile, false)
 				count := e2e.CountOfStringInSlice("test-daemonset", pods)
-				fmt.Println("POD COUNT")
-				fmt.Println(count)
-				fmt.Println("NODE COUNT")
-				fmt.Println(len(nodes))
 				g.Expect(len(nodes)).Should((Equal(count)), "Daemonset pod count does not match node count")
 			}, "420s", "10s").Should(Succeed())
 		})
 
+		It("Verifies Restart K3s", func() {
+			_, err := e2e.DeployWorkload("daemonset.yaml", kubeConfigFile, *hardened)
+			Expect(err).NotTo(HaveOccurred(), "Daemonset manifest not deployed")
+			nodes, _ := e2e.ParseNodes(kubeConfigFile, false)
+
+			Eventually(func(g Gomega) {
+				pods, _ := e2e.ParsePods(kubeConfigFile, false)
+				count := e2e.CountOfStringInSlice("test-daemonset", pods)
+				g.Expect(len(nodes)).Should((Equal(count)), "Daemonset pod count does not match node count")
```

Comment:
Can we also add an expectation here to ensure that the daemonset pods are Running? This so far just validates that they are deployed to the nodes, but they could be Crashlooping